src/octopeer-github/content/EventTracker/MousePositionTracker.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Created by Mathias on 2016-05-27.
+ */
+class MousePositionTracker {
+
+    /** Private static final for the timeout between logs. */
+    private static get MOUSE_POSITION_TRACKER_TIMEOUT() { return 500; }
```

Comment:
Since the class is called MousePositionTracker, `MousePositionTracker.MOUSE_POSITION_TRACKER_TIMEOUT` looks a bit redundant. Can you rename this to `TIMEOUT`?